src/mlpack/tests/CMakeLists.txt

```diff
@@ -232,7 +232,6 @@ set(parallel_tests
   "GMMTest;"
   "CFTest;"
   "HMMTest;"
-  "LARSTest;"
```

Comment:
I think we will want to have this list of parallel tests back later; maybe that's worth adding to the issue description just so we don't forget?

Reply:
@zoq , Can you help us with this ?

Reply:
I agree with @rcurtin, let's update the issue description mention that we should compare the test runtime with the boost runtime once the transition is complete, if the difference is huge let's find a solution to run parallel tests.